docs/StepbyStepManualDeploymentGuideScalarDLOnAzure.md

```diff
@@ -0,0 +1,237 @@
+
+* Select [Virtual networks](https://portal.azure.com/#blade/HubsExtension/BrowseResource/resourceType/Microsoft.Network%2FvirtualNetworks) from Azure portal
+* Click **Create**
+* On the Basics page, configure the following options
+    * Select your **Subscription**
+    * Select the **Resource group**
+    * Enter **Name**
+    * Select **Region**
+    * Click **Next: IP Addresses**
+* On the IP Addresses page, enter the following values for the virtual network.
+    * Enter **IPv4 Address space** as _10.42.0.0/16_ for virtual network.
+    * Click **Add subnet** and add the following subnets
+        * Enter **Subnet name** as _public_ and **subnet address range** as _10.42.0.0/24_ then click **Add**
+        * Enter **Subnet name** as _k8s_node_pod_ and **subnet address range** as _10.42.40.0/22_ then click **Add**
+        * Enter **Subnet name** as _k8s_ingress_ and **subnet address range** as _10.42.44.0/22_ then click **Add**
+    * Click **Review+create**
+    * Click **Create**
+
+## Create Cosmos DB
+
+You can use following steps to create Cosmos DB account based on the basis of [requirements](./SetupAzureDatabase.md#cosmos-db)
+
+* Select [Cosmos DB](https://portal.azure.com/#blade/HubsExtension/BrowseResource/resourceType/Microsoft.DocumentDb%2FdatabaseAccounts) from Azure portal
+* Click **Create**
+* Click **Create** in _Core (SQL) - Recommended_
+* Select your **Subscription**
+* Select your **Resource group**
+* Enter **Account Name**
+* Select **Location**
+* Select **Capacity mode** as _Provisioned throughput_
+* Click **Review+Create**
+* Click **Create**
+* Click **Go to resource** (option available after Cosmos DB account creation)
+* Select **Default consistency** from left navigation
+    * Select _STRONG_ and click **Save**
+
+## Create Bastion
+
+You must have a bastion server to access the private AKS cluster. You can use the following steps to create a bastion server.
+ 
+* Select [Virtual machines](https://portal.azure.com/#blade/HubsExtension/BrowseResource/resourceType/Microsoft.Compute%2FVirtualMachines) from Azure portal
+* Click **Create**
+* Select **Virtual machine**
+* On the Basics page, configure the following options
+    * Select an Azure **Subscription**
+    * Select your **Resource group**
+    * Enter **Virtual machine name**
+    * Select **Region**
+    * Select any of the following availability options in the dropdown menu, select any of the following availability options in the dropdown menu, You can select the _No infrastructure redundancy required_ option if you don't want any.
+        * Availability set: Select created availability set from dropdown menu
+        * Availability zone: Select an availability zone 1 or 2 or 3
+    * Select Image _CentOS-based 7.9 - Gen 1_
+    * Select **Size** as _Standard_B1s_ from dropdown menu
+    * Select authentication type as SSH public key(generate an SSH key to access the server) or password(can use username and password to SSH access the server).
```

Comment:
Password authentication is not recommended. so I don't think it needs to be write here. what do you think?

Reply:
Yes, we can remove that, removed it in 8f9b0ff